### PR TITLE
Vendorize language pages

### PR DESCRIPTION
**Pull request by chadwcarlson**
## What's changed

Upsun & white-label vendorization for all pages within languages.

- [x] go
- [x] lisp
- [x] ruby
- [x] rust
- [x] elixir
- [x] dotnet
- [x] python
- [x] php
- [x] nodejs
- [x] java

**Blocked by "Manage resources" https://github.com/platformsh/platformsh-docs/pull/3434**

### CODE REVIEW

File: sites/platform/src/languages/java/migration.md [hunk 1]
```diff
@@ -31,26 +31,37 @@ For more details, see [tuning](./tuning.md).
 ## Monolith/Single Application
 
 To start a Java application, you need to understand the [{{% vendor/name %}} structure](/learn/overview/structure.md).
-At minimum, you need to configure your [application](../../create-apps/_index.md).
-You can also have two [YAML files](/learn/overview/yaml/_index.md)  if you need:
-
-* [Routes](../../define-routes/_index.md)
-* [Services](../../add-services/_index.md)
+You will need to configure your [application](../../create-apps/_index.md), [routes](../../define-routes/_index.md),
+and [services](../../add-services/_index.md)
```
Comment (AnouckColson) [inline sites/platform/src/languages/java/migration.md:35]:
```suggestion
and [services](../../add-services/_index.md).
```

File: sites/platform/src/languages/java/migration.md [hunk 2]
```diff
@@ -63,23 +74,51 @@ Be aware that after the build, it creates a read-only system. You have the [moun
...
-The largest available is 8 GB.
+{{< note version="1" >}}
+Application instances have a limited amount of memory at build time, which has a maximum of 8 GB.
+At runtime that limit depends on your plan and configuration.
 A stateless application can be scaled horizontally to multiple application instances using Varnish in a [load balancer](https://community.platform.sh/t/how-to-configure-load-balancer-in-a-single-application/553) configuration.
 {{< /note >}}
 
+{{< note version="2" >}}
+Application instances have a limited amount of memory at build time, which has a maximum of 8 GB.
+At runtime that limit depends on the resources you have defined for your application container using `{{% vendor/cli %}} resources:set`.
```
Comment (AnouckColson) [inline sites/platform/src/languages/java/migration.md:118]:
```suggestion
At runtime that limit depends on [the resources you have defined for your application container](/manage-resources.md) using `{{% vendor/cli %}} resources:set`.
```

File: sites/platform/src/languages/java/migration.md [hunk 3]
```diff
@@ -100,17 +139,26 @@ You have the option to use several languages in microservices. If you're using J
 | [Multiple Applications](https://community.platform.sh/t/multiple-applications-tomcat/468) | [Source](https://github.com/platformsh-examples/tomcat-multi-app) |
 | [Configure multi-applications with `{{< vendor/configfile "apps" >}}`](https://community.platform.sh/t/how-to-configure-multi-applications-with-applications-yaml/552) | [Source](https://github.com/platformsh-examples/tomcat-multi-app-applications) |
 
-{{< note >}}
+{{< note version="1" >}}
+
+As a single application, in the multi-app, you have the option to set load balancer to some or [all applications in the project cluster](https://community.platform.sh/t/how-to-configure-load-balancer-in-a-multiple-applications/554).
```
Comment (AnouckColson) [inline sites/platform/src/languages/java/migration.md:144]: As for, or as with?

File: sites/platform/src/languages/php/xdebug.md [hunk 4]
```diff
@@ -33,11 +33,22 @@ Xdebug runs as a second PHP-FPM process used only for debugging requests, leavin
 
 To enable Xdebug, add the following to your [app configuration](../../create-apps/app-reference.md):
 
+{{% version/specific %}}
```
Comment (AnouckColson) [inline sites/platform/src/languages/php/xdebug.md:36]: l. 20: mention to Grid needs to be removed for Upsun.

**Review by AnouckColson — CHANGES_REQUESTED** [review]

File: sites/platform/src/languages/java/migration.md (hunk 3 above)
Comment (chadwcarlson) [reply sites/platform/src/languages/java/migration.md]: Yeah that is bad. Fixed

**Review by AnouckColson — APPROVED** [review]

**Pull request merged**